### DISCUSSION TITLE
accessibility of proof

### DISCUSSION THREAD

**Discussion by lgilnadolskis** [community/community#134906] [GitHub Education]
### Hi everyone,

Hello.
As a visually impaired student, the process to submit proof of eligibility is very frustating.
I have tried several times and I keep getting an error.

I am not sure if the issue is on the pictures themselves or something else.

**Comment by everforyou** [comment]
Hi,
To address the issue you are experiencing with submitting proof of eligibility, I recommend the following professional steps

Ensure that the images you are attempting to upload meet the required specifications, including file format (e.g., JPEG, PNG) and size limitations.

As a visually impaired student, consider using specialized accessibility software or tools that can assist you in navigating the submission process more effectively.

^-^

**Comment by lgilnadolskis** [comment]
The file I am submitting is a high definition picture of my student ID.

**Comment by everforyou** [comment]
Hey there! 😊

1. **Check the Specs**: Make sure the image meets any requirements, like the right file format (JPEG or PNG) and size limits. Sometimes, those little details can make a big difference!

2. **Accessibility Tools**: Since you mentioned being visually impaired, using some specialized accessibility software might help you navigate the submission process more smoothly.

3. **Troubleshooting**: If you’re still running into issues, it could be worth checking if the image is too large or if there are any specific guidelines from the platform.

I hope this helps, and best of luck with your submission! If you have any more questions, feel free to ask. 🌟

^_^
